DESCRIPTION

```diff
@@ -50,6 +50,7 @@ Imports:
     GO.db,
     org.Hs.eg.db,
     Matrix,
+    matrixTests,
```

Comment:
Tara, what is the library for?

Reply:
This is for parallelizing the statistical test for each edge in the network. Doing them one-by-one makes the process very slow, so I used matrixTests to do them all at once.

Reply:
Great!